### DISCUSSION TITLE
Compatible with nrf24l01

### DISCUSSION THREAD

**Discussion by poloalex** [Yogui79/IntexPureSpa#29] [Q&A]
Hi,
I have a nrf24l01 chip instead of the lc12s. I think it is compatible but I guess I have to make some changes... 
Do you know how to wire it correctly to the esp32 and what to change in the code to make it work ? 

Thanks

Zethy

**Comment by Yogui79** [comment]
Hello, 
know if it compatible becose the LC12s is make to do direct conection not like a NRF24l01

**Comment by djgel** [comment]
I believe it is possible, even using it in ESP8266, as it uses the SPI interface for communication, but it would be necessary to change the codes completely! But you will have to change the code by 30%

**Reply by Yogui79** [reply]
you need to complet change the communication yes over spi I don't know how to handel the networtk id with the NRF

**Comment by poloalex** [comment]
Okay thanks ! I think I will use LCs (I ordered it) becuase I don't know how to find channel ID and Network ID with the NFR even if I rewrite code (there is no LED on NRF) !
